output/elastic/outputelastic.go

```diff
@@ -22,7 +22,8 @@ const ModuleName = "elastic"
 // OutputConfig holds the configuration json fields and internal objects
 type OutputConfig struct {
 	config.OutputConfig
-	URL             []string `json:"url"`               // elastic API entrypoints
+	URL             []string `json:"url"` // elastic API entrypoints
```

Comment:
We should align comment here

Reply:
the current format is what `go fmt` produces. If I run `go fmt` again I see no changes and if I modify manually and run `go fmt`again the changes are undone.

Are you suggesting to align all comments in the OutputConfig struct?

I can do that for sure, just that I don't think the code will pass `go fmt` as before